bitsandbytes/functional.py

```diff
@@ -2517,3 +2535,57 @@ def pipeline_test(A, batch_size):
     out = torch.zeros_like(A)
     lib.cpipeline_test(get_ptr(A), get_ptr(out), ct.c_size_t(A.numel()), ct.c_size_t(batch_size))
     return out
+
+
+# 8 bits functions
+def double_quant(A, col_stats=None, row_stats=None, out_col=None, out_row=None, threshold=0.0):
+        if A.device == "cuda":
+            return cuda_double_quant(A=A, col_stats=col_stats, row_stats=row_stats, out_col=out_col, out_row=out_row, threshold=threshold)
+        else:
+            raise RuntimeError("double_quant on non-CUDA devices (CPU, XPU...) will be soon supported but not yet, aborting...")
```

Comment:
No need to promise in the error message.
```suggestion
            raise RuntimeError("double_quant is not supported on non-CUDA devices")
```